conf/messages

```diff
@@ -33,102 +19,71 @@ image.create.failed=Failed to create image
+user.lastAdmin=Your account is the last remaining admin in your organzation. You cannot remove admin privileges from your account.
 
-user.resetPassword.failed=Invalid old password
-user.resetPassword.success=Your password was successfully changed
 user.email.alreadyInUse=This email address is already in use
 user.email.onlySuperUserCanChange=Only super users can change emails of users with multiple organization accounts
-user.firstName.empty=Please enter your first name
-user.lastName.empty=Please enter your last name
 user.firstName.invalid=Please check your first name for any special characters
 user.lastName.invalid=Please check your last name for any special characters
-user.login.failed=Invalid email or password
-user.login.success=You are now logged in
-user.logout.success=You have successfully logged out
-user.configuration.invalid=Couldn’t parse configuration
+user.configuration.invalid=Could not parse configuration
 user.configuration.updated=Your configuration was updated
 user.configuration.dataset.invalid=Dataset configuration is invalid
 user.configuration.dataset.updated=Dataset configuration updated
-user.experience.set=Set experience of {0}
-user.experience.removed=Removed experience from {0}
 user.email.invalid=At least one Email is invalid
 user.notAuthorised=You are not authorized to view this resource. Please log in.
-user.id.notFound=We couldn’t find a user id in the request.
+user.id.notFound=We could not find a user id in the request.
 user.id.invalid=The provided user id is invalid.
 user.creation.failed=Failed to create user
 
 braintracing.new=An account on braintracing.org was created for you. You can use the same credentials as on webKnossos to login.
-braintracing.error=We couldn’t atomatically create an account for you on braintracing.org. Please do it on your own.
+braintracing.error=We could not atomatically create an account for you on braintracing.org. Please do it on your own.
 braintracing.exists=Great, you already have an account on braintracing.org. Please double check that you have uploaded all requested information.
 
 dataSet=Dataset
-dataSet.file=Dataset ZIP File
 dataSet.notFound=Dataset {0} does not exist or could not be accessed
 dataSet.notFoundConsiderLogin=Dataset {0} does not exist or could not be accessed. You may need to log in.
 dataSet.notFoundForAnnotation=The Dataset for this annotation does not exist or could not be accessed.
 dataSet.noAccess=Could not access DataSet {0}. Does your team have access?
 dataSet.noAccessById=Could not access the corresponding DataSet. This is likely because you are not a member of a team that has access to it.
 dataSet.notImported=Dataset {0} is not imported
-dataSet.name.invalid=The dataset name is invalid
-dataSet.name.invalid.characters=Dataset names may only contain letters, digits, dots, underscorees, dashes.
+dataSet.name.invalid=The dataset name contains illegal characters. Please only use letters, numbers, and hypens.
```

Comment:
This is not always the case, one of the three messages below should be in the chain. We may want to remove this message entirely so the ones from below can bubble up to the main message

Reply:
I am not sure what you want me to do here? None of the messages were removed. I just rephrased them.